scheduler/core/builder/blueprint.py

```diff
@@ -13,8 +13,8 @@
 
 from scheduler.config import config, ConfigurationError
 from scheduler.core.components.optimizer.dummy import DummyOptimizer
-from scheduler.external.resource import ResourceMock
-from scheduler.external.environment import Env
+from scheduler.services.resource import OcsResourceService
```

Comment:
I don't really see any way around it, given that the information will be coming from completely different sources. Perhaps we can put a layer between `OcsResourceService` and `GppResourceService`, but right now, since I don't think there is a `GppResourceService`, it makes much sense to think about that now.

Something that I thought about that could get very confusing: if someone requests plan generation that begins before OCS becomes inactive and ends when GPP becomes active.

Reply:
In fact, since the mock is gone, we can probably remove this completely from the builder / blueprint / config.

Reply:
> I don't really see any way around it, given that the information will be coming from completely different sources. Perhaps we can put a layer between `OcsResourceService` and `GppResourceService`, but right now, since I don't think there is a `GppResourceService`, it makes much sense to think about that now.
> 
> Something that I thought about that could get very confusing: if someone requests plan generation that begins before OCS becomes inactive and ends when GPP becomes active.

Yes I think this might be an issue for those special cases.